scripts/exregional_nonvarcldanl.sh

```diff
@@ -237,7 +237,7 @@ cat << EOF > gsiparm.anl
    r_cleanSnow_WarmTs_threshold=5.0,
    l_conserve_thetaV=.true.,
    i_conserve_thetaV_iternum=3,
-   l_cld_bld=.true.,
+   l_cld_bld=.false.,
```

Comment:
Do other applications need to turn off the cloud build?

Reply:
This commit is to save what I have for enabling the cloud analysis for the ensemble runs, where we were trying to have cloud clearing only (l_cld_bld=.false.). But the cloud analysis itself might not be ready yet, at least from my ensemble runs. So when it is really ready, we can always change this.

Reply:
@chunhuazhou We need try to avoid committing this kind of hardwired change only used for certain application. We will forgot it and other cloud analysis tests will use it for a while before someone notice difference. Please set up it up through the configuration step.

Reply:
This is in gsiparm.anl. I think it should be set in the config file instead of in the script.

Reply:
@chunhuazhou If you check ush/config.sh.3DRTMA_dev1, there is a section "GSI Namelist parameters configurable across differnt applications". You can follow those examples to make i_cld_bld configurable.

Reply:
Agreed. I overlooked this change when committing it. 
And this is not the same as the gsiparm.anl as for the GSI analysis so it is a bit different here. 
Should I open another PR to address this and that "ENS_TYPE" thing? I merged this PR too quickly.

Reply:
Opening a new PR works.

Reply:
Ok, will do. Thanks!

Reply:
> @chunhuazhou We need try to avoid committing this kind of hardwired change only used for certain application. We will forgot it and other cloud analysis tests will use it for a while before someone notice difference. Please set up it up through the configuration step.

@hu5970 I have a similar concern on you potential manual changing "ln_vrfy" to "cp_vry" in realtime RRFS runs. Only you will remember that. It will lose change history track and will be easily ignored by others. My suggestion is to let this be configurable through the existed "do_nonvar_cldanal" option.